Variable.java

```diff
@@ -0,0 +1,15 @@
+package Homework;
+
+public class Variable {

```

Comment:
I like the use of a class here! Though Variable is a bit of a vague name, you could name it something like Triplet for example that is more indicative of what it is